### PR TITLE
[Config] Add default configuration of load_parallelism

### PR DESCRIPTION
**Pull request by huangmengbin**
## Proposed changes

- Make load_parallelism configurable. 

- Different clusters should be configured with different load_parallelism values.
- Some user don't know how to set load_parallelism, or don't know the best load_parallelism value.

## Types of changes

configuration

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

Thanks!

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -128,6 +128,13 @@
     @ConfField(mutable = true, masterOnly = true)
     public static int label_keep_max_second = 3 * 24 * 3600; // 3 days
 
+    /**
+     * The default parallelism of the load execution plan
+     * on a single node when the broker load is submitted
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static int load_parallelism = 1;
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:136]:
```suggestion
    public static int default_load_parallelism = 1;
```

File: docs/zh-CN/administrator-guide/config/fe_config.md [hunk 2]
```diff
@@ -1049,6 +1049,16 @@ colocote join PlanFragment instance 的 memory_limit = exec_mem_limit / min (que
...
 
+### load_parallelism
+
+默认值：1
+
+是否可以动态配置：true
+
+是否为 Master FE 节点独有的配置项：true
+
+单个节点导入的默认并发度
```
Comment (EmmyMiao87) [inline docs/zh-CN/administrator-guide/config/fe_config.md:1060]: 首先这里要强调，这是一个默认的导入并发度。如果用户指定自定义的并发度，则使用自定义并发度。
其次导入任务的并发度取决于多个参数，这只是其中之一，参见 BrokerScanNode 的 line 355 ~ 362

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java (hunk 1 above)
Comment (huangmengbin) [reply fe/fe-core/src/main/java/org/apache/doris/common/Config.java]: done.

File: docs/zh-CN/administrator-guide/config/fe_config.md (hunk 2 above)
Comment (huangmengbin) [reply docs/zh-CN/administrator-guide/config/fe_config.md]: 是的，我们团队也留意到了那段代码，不过之前没有写入文档之中。感谢你的指正！不知道新的写法是否ok？

**Review by morningman — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
@EmmyMiao87 FYI

**Pull request merged**